docs/updates/migrate-v3.md

```diff
@@ -129,7 +129,7 @@ The following processors have been removed:
 
 ## Removal of Standard Middleware
 
-The following middlewares have been removed:
+The following middlewares have been removed from the [standard middlewares list](../../pages/Server/#standard-middleware):
```

Comment:
its one `../` to much isn't?

Reply:
Unfortunately not, the migration path is `https://sap.github.io/ui5-tooling/v3/updates/migrate-v3/` and the server `https://sap.github.io/ui5-tooling/v3/pages/Server/`, so basically the two `../` would go down to `https://sap.github.io/ui5-tooling/v3/` and then append `/pages/Server/#standard-middleware`

Reply:
Ack